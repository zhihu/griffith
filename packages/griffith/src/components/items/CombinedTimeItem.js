import React from 'react'
import {css} from 'aphrodite/no-important'
import styles from '../Controller.styles'
import Time from '../Time'

const CombinedTimeItem = ({isFullScreen, currentTime, duration}) => (
  <div className={css(styles.time, isFullScreen && styles.fullScreenedTime)}>
    <div className={css(styles.timeText)}>
      <Time value={currentTime} />
      {' / '}
      <Time value={duration} />
    </div>
  </div>
)

export default CombinedTimeItem
