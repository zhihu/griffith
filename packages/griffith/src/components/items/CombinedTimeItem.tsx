import React from 'react'
import {css} from 'aphrodite/no-important'
import styles from '../Controller.styles'
import Time from '../Time'

const CombinedTimeItem = ({isFullScreen, currentTime, duration}: any) => (
  <div className={css(styles.time, isFullScreen && styles.fullScreenedTime)}>
    <div>
      <Time value={currentTime} />
      {' / '}
      <Time value={duration} />
    </div>
  </div>
)

export default CombinedTimeItem
