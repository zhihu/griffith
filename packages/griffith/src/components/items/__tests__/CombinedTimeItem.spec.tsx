import React from 'react'
// @ts-expect-error ts-migrate(7016) FIXME: Could not find a declaration file for module 'enzy... Remove this comment to see the full error message
import {shallow} from 'enzyme'
import CombinedTimeItem from '../CombinedTimeItem'

describe('CombinedTimeItem', () => {
  it('get CombinedTimeItem component', () => {
    const wrapper = shallow(
      <CombinedTimeItem isFullScreen={true} currentTime={6} duration={182} />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
