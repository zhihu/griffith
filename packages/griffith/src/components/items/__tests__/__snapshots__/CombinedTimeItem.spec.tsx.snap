// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CombinedTimeItem get CombinedTimeItem component 1`] = `
<div>
  <div class="time_1n8zam0-o_O-fullScreenedTime_1xxtrer">00:06 / 03:02</div>
</div>

`;
